data catalog generate-site failing because the prefix for owl could not be found

Discovered on the [feature/use-local-model-xlsx](https://bitbucket.pearson.com/projects/METIS/repos/metis-onecrm/browse?at=refs%2Fheads%2Ffeature%2Fuse-local-model-xlsx) branch of the onecrm-metis project in Pearson bitbucket.

The konig-data-catalog-maven-plugin generate-site goal is failing because it cannot find the prefix for owl.  The rdf dir that the plugin is reading does include an owl.ttl file generated from the spreadsheet (shown below). 

**error:**
```
[ERROR] Failed to execute goal io.konig:konig-data-catalog-maven-plugin:2.0.0-8-SNAPSHOT:generate-site (default) on project data-catalog: Execution default of goal io.konig:konig-data-catalog-maven-plugin:2.0.0-8-SNAPSHOT:generate-site failed: Prefix not found for namespace http://www.w3.org/2002/07/owl# -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.konig:konig-data-catalog-maven-plugin:2.0.0-8-SNAPSHOT:generate-site (default) on project data-catalog: Execution default of goal io.konig:konig-data-catalog-maven-plugin:2.0.0-8-SNAPSHOT:generate-site failed: Prefix not found for namespace http://www.w3.org/2002/07/owl#
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:213)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:51)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:309)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:194)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:107)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:955)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default of goal io.konig:konig-data-catalog-maven-plugin:2.0.0-8-SNAPSHOT:generate-site failed: Prefix not found for namespace http://www.w3.org/2002/07/owl#
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:145)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:51)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:309)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:194)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:107)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:955)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
Caused by: io.konig.core.KonigException: Prefix not found for namespace http://www.w3.org/2002/07/owl#
    at io.konig.shacl.ClassStructure$Builder.shapeName (ClassStructure.java:899)
    at io.konig.shacl.ClassStructure$Builder.produceShape (ClassStructure.java:880)
    at io.konig.shacl.ClassStructure$Builder.injectThingAndNullShape (ClassStructure.java:598)
    at io.konig.shacl.ClassStructure$Builder.build (ClassStructure.java:348)
    at io.konig.shacl.ClassStructure$Builder.access$000 (ClassStructure.java:334)
    at io.konig.shacl.ClassStructure.init (ClassStructure.java:122)
    at io.konig.shacl.ClassStructure.<init> (ClassStructure.java:85)
    at io.konig.datacatalog.DataCatalogBuilder.build (DataCatalogBuilder.java:112)
    at io.konig.maven.datacatalog.KonigDataCatalogMojo.execute (KonigDataCatalogMojo.java:99)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:134)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:51)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:309)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:194)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:107)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:955)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)

```

rdf/owl/owl.ttl:
```
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix vann: <http://purl.org/vocab/vann/> .

owl: a owl:Ontology ; 
	vann:preferredNamespacePrefix "owl" ; 
	rdfs:label "OWL" ; 
	rdfs:comment "Web Ontology Language" . 

owl:Thing a owl:Class . 
```

--- a/konig-core/src/main/java/io/konig/shacl/ClassStructure.java
+++ b/konig-core/src/main/java/io/konig/shacl/ClassStructure.java
@@ -30,6 +30,7 @@ import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import io.konig.core.*;
 import org.openrdf.model.Namespace;
 import org.openrdf.model.Resource;
 import org.openrdf.model.URI;
@@ -40,11 +41,6 @@ import org.openrdf.model.vocabulary.RDFS;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import io.konig.core.Graph;
-import io.konig.core.KonigException;
-import io.konig.core.OwlReasoner;
-import io.konig.core.Path;
-import io.konig.core.Vertex;
 import io.konig.core.impl.RdfUtil;
 import io.konig.core.util.SimpleValueMap;
 import io.konig.core.util.ValueFormat;
@@ -117,7 +113,9 @@ public class ClassStructure {
 	}
 	
 	public void init(ShapeManager shapeManager, OwlReasoner reasoner) {
-
+		if(reasoner != null && reasoner.getGraph() != null && reasoner.getGraph().getNamespaceManager() != null) {
+			reasoner.getGraph().getNamespaceManager().add("owl", "http://www.w3.org/2002/07/owl#");
+		}
 		Builder builder = new Builder(shapeManager, reasoner);
 		builder.build();
 	}
@@ -894,6 +892,7 @@ public class ClassStructure {
 		private URI shapeName(URI targetClassURI) {
 			SimpleValueMap map = new SimpleValueMap();
 			map.put("targetClassLocalName", targetClassURI.getLocalName());
+//			reasoner.getGraph().getNamespaceManager().add("owl", "http://www.w3.org/2002/07/owl#");
 			Namespace ns = reasoner.getGraph().getNamespaceManager().findByName(targetClassURI.getNamespace());
 			if (ns == null) {
 				throw new KonigException("Prefix not found for namespace " + targetClassURI.getNamespace());

